src/build/app/mod.rs

```diff
@@ -180,7 +180,7 @@ impl<'help> App<'help> {
 
     /// Iterate through the set of *all* the aliases for this subcommand, both visible and hidden.
     #[inline]
-    pub fn get_all_aliases(&self) -> impl Iterator<Item = &str> {
+    pub fn get_all_aliases(&self) -> impl Iterator<Item = &str> + '_ {
```

Comment:
&'help str

Reply:
I ran into problems with lifetimes with `all_subcommand_names` and didn't quickly resolve it.  Rather than old this up, I punted on it for now.  Sorry I forgot to comment on that before.